### ISSUE TITLE
MAIL_URL=smtp  >>  'ATCreateUserServer' Error: getaddrinfo ENOTFOUND

### ISSUE THREAD

**Issue by jujes** [wekan/wekan#898]
Hi, some one know how fix it?

my docker-compose.yml  environment is:

```
      - MAIL_URL=smtp://XXXXX%40gmail.com:YYYYY@smtp.gmail.com:465/
      - MAIL_FROM=XXXXX@gmail.com
```

output
```
wekan-app  | Exception while invoking method 'ATCreateUserServer' Error: getaddrinfo ENOTFOUND
wekan-app  |     at Object.Future.wait (/build/programs/server/node_modules/fibers/future.js:449:15)
wekan-app  |     at smtpSend (packages/email/email.js:89:1)
wekan-app  |     at Object.Email.send (packages/email/email.js:192:1)
wekan-app  |     at AccountsServer.Accounts.sendVerificationEmail (packages/accounts-password/password_server.js:797:9)
wekan-app  |     at [object Object].Meteor.methods.ATCreateUserServer (packages/useraccounts_core/lib/server_methods.js:119:1)
wekan-app  |     at packages/check/match.js:107:1
wekan-app  |     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
wekan-app  |     at Object.exports.Match._failIfArgumentsAreNotAllChecked (packages/check/match.js:106:1)
wekan-app  |     at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1708:18)
wekan-app  |     at packages/ddp-server/livedata_server.js:711:19
wekan-app  |     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
wekan-app  |     at packages/ddp-server/livedata_server.js:709:40
wekan-app  |     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
wekan-app  |     at packages/ddp-server/livedata_server.js:707:46
wekan-app  |     at tryCallTwo (/build/programs/server/npm/node_modules/meteor/promise/node_modules/promise/lib/core.js:45:5)
wekan-app  |     at doResolve (/build/programs/server/npm/node_modules/meteor/promise/node_modules/promise/lib/core.js:200:13)
wekan-app  |     - - - - -
wekan-app  |     at errnoException (dns.js:37:11)
wekan-app  |     at Object.onanswer [as oncomplete] (dns.js:126:16)
```

**Comment by pascal20997** [comment]
Got the same problem with SMTP as environment variables. I think the best solution is setting the mail settings via the new admin panel (https://github.com/wekan/wekan/blob/devel/CHANGELOG.md). This worked for me like a charm. Please notice: Remove the environment variable before/after you set the settings via admin panel otherwise it won´t work.

**Comment by jujes** [comment]
Hi @pascal20997 I tried with the new setting feature but ir don't work... 
here my docker-compose.yml maybe you see the issue :)

I use:  **docker-compose up -d**

thanks in advance,

```
version: '2'

services:

  wekandb:
    image: mongo:3.2.11
    container_name: wekan-db
    restart: always
    command: mongod --smallfiles --oplogSize 128
    networks:
      - wekan-tier
    expose:
      - 27017
    volumes:
      - wekan-db:/data/db
      - wekan-db-dump:/dump

  wekan:
    image: mquandalle/wekan:latest
    container_name: wekan-app
    restart: always
    networks:
      - wekan-tier
    build:
      context: .
      dockerfile: Dockerfile
      args:
        - NODE_VERSION=${NODE_VERSION}
        - METEOR_RELEASE=${METEOR_RELEASE}
        - NPM_VERSION=${NPM_VERSION}
        - ARCHITECTURE=${ARCHITECTURE}
        - SRC_PATH=${SRC_PATH}
    ports:
      - 80:80
    environment:
      - MONGO_URL=mongodb://wekandb:27017/wekan
      - ROOT_URL=http://wekan.dev
    depends_on:
      - wekandb

volumes:
  wekan-db:
    driver: local
  wekan-db-dump:
    driver: local

networks:
  wekan-tier:
    driver: bridge

```

**Comment by max-marschall** [comment]
That issue is related to an Error message after registration, but after trying to login: everything is working.

![error-registration](https://cloud.githubusercontent.com/assets/26233216/23829506/67def32e-06f3-11e7-8861-89394aba4029.JPG)

**Comment by max-marschall** [comment]
Sure @jujes, I just tried my stuff :)

**Comment by REJack** [comment]
i had issues too with the smtp setting, i realized my password was the problem for the url. 
some special chars cause errors (like @, & and %)

**Comment by xet7** [comment]
@jujes 

Does this also happen on today's Wekan release v0.13 ?

**Comment by xet7** [comment]
@jujes 

ENOTFOUND usually means that there is already node running when you are starting another node.

Anyway, this looks similar to bug that was fixed in v0.13, so I'm closing this issue.
